docs/topics/api/auth_internal.rst

```diff
@@ -39,6 +39,35 @@ The token is available in two forms:
     * For all endpoints, as a cookie called ``api_auth_token``. This cookie
       expires after 30 days and is set as ``HttpOnly``.
 
+The response will contain some profile data for personalization:
+
+    :>json int id: The numeric user id.
+    :>json string email: Email address used by the user to log in and create this account.
+    :>json string name: The name chosen by the user, or the username if not set.
+    :>json string picture_url: URL to a photo of the user, or `/static/img/anon_user.png` if not set.
+    :>json string username: username chosen by the user, used in the account url. If not set will be a randomly generated string.
+    :>json array permissions: A list of the additional :ref:`permissions <login-response-permissions>` this user has.
+
+.. _login-response-permissions:
+
+    Permissions can be any arbritary string in the format `app:action`. Either `app` or `action` can be
+    the wildcard `*`, so `*:*` means the user has permission to do all actions (i.e. full admin).
```

Comment:
Does this mean we will get permissions like `Addons:*` rather than a list of all relevant permissions? That's more work on the front-end as we'll need to check for both `Addons:*` and `Addons:edit`... but I guess it's efficient from the API's end rather than sending a huge list of permissions and we'll probably use a utility function to check anyway, so actually that's fine.